### ISSUE TITLE
If the input has `?` in the end it crashed

### ISSUE THREAD

**Issue by MohamedAliRashad** [AMR-KELEG/English-to-arabic-transphonator#3]
```python
trans = Transphonator()
english_sentence = "Who the hell do you think you are ?"
print(" ".join([trans.transphonate_english_word(word) for word in english_sentance.split()]))

```
```error
TypeError: sequence item 8: expected str instance, NoneType found
```

**Comment by AMR-KELEG** [comment]
I agree, this is a nasty bug caused by not handling words that are not part of the dictionary.
https://github.com/AMR-KELEG/English-to-arabic-transphonator/blob/86412b43f9b3d686a0acca3f0a2d65537181a46f/english_to_arabic_transphonator/transphonator.py#L44-L49
I would say at least it should return the word as a fallback. Better solutions would also raise a warning if the word has latin characters (i.e.: If a word matches this regular expression `r"[a-zA-Z]+"`), and do nothing if the word is just a set of punctuation marks.
Are you interested in fixing this bug?

**Comment by MohamedAliRashad** [comment]
defaulting to the original word is a good and easy fix
